.github/actions/before-install/action.yml

```diff
@@ -0,0 +1,14 @@
+name: "before-install"
+description: "launch before_install.sh and then parse env vars"
+runs:
+  using: "composite"
+  steps:
+    - name: Before install
+      shell: bash
+      run: |
+        export S3_DBP_PATH='s3://gh-action-builds/adf'
```

Comment:
should there be a notion of DBP in the ADF?

Reply:
it was already like that on travis not a big issue